tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -153,7 +153,7 @@ spec:
                 ORAS_ARGS=()
                 if [[ "$MEDIA_TYPE" == "application/vnd.docker.distribution.manifest.list.v2+json" ]]\
                   || [[ "$MEDIA_TYPE" == "application/vnd.oci.image.index.v1+json" ]]; then
-                  ORAS_ARGS+=("--platform $os/$arch")
+                  ORAS_ARGS+=("--platform $OS/$ARCH")
```

Comment:
can we add a test here?

Reply:
Yeah, but if this passes e2e before I have it, the test will be a separate PR because this is breaking users